### DISCUSSION TITLE
Play List - formats supported

### DISCUSSION THREAD

**Discussion by Joepic9** [music-assistant/support#1088] [Q&A] [answered]
### PLAY LIST
1. What Play List formats does MA support?  See list below.
2. Does MA support a Play List that references different folders or must all songs be in the same folder as the Play List?
3. Here is an example of a Play List (created with CLEMENTINE) that has the music in a different folder from the Play List.
    #EXTM3U
    #EXTINF:176, - A Fistful of Dollars Theme (Ennio Morricone)
    //ds1815/music/4K Video Downloader/A Fistful of Dollars Theme (Ennio Morricone).mp3
4. What is the quickest way for MA to see a new Play List that I just created?  Must I do a Run Sync on just the Play List or All folders?
5. What is the best way for MA to see a new song/folder that I have added?
6. Software called **CLEMENTINE** is a good way to generate Play List.

Thanks
David - WA8KHP

![playlist](https://user-images.githubusercontent.com/121582698/216841795-13771ef9-5425-47b1-8120-2cba1a3944cf.jpg)

**Comment by OzGav** [comment, marked as answer]
M3U, M3U8, PLS
